### PR TITLE
Add Paraview 5.7.0

### PR DESCRIPTION
**Pull request by jrood-nrel**

### CODE REVIEW

**Comment by scheibelp** [comment]
Hi @jrood-nrel it looks like this triggered a secondary check: with https://github.com/spack/spack/pull/11115 packages are required to implement:

```
setup_dependent_build_environment(self, env)

setup_dependent_run_environment(self, env)
  #setup_dependent_environment only modifies spack_env so you should not have to implement this

setup_build_environment(self, env)
  #setup_environment only modifies run_env so you should not have to implement this

setup_run_environment(self, env)
```

vs.

```
setup_environment(self, spack_env, run_env)

setup_dependent_environment(self, spack_env, run_env)
```

would you be able to do that?

**Comment by alalazo** [comment]
Hi @jrood-nrel let me know if you need any help with respect to fixing the issue above.

**Comment by jrood-nrel** [comment]
I think I understand. I made an update.

File: var/spack/repos/builtin/packages/paraview/package.py [hunk 1]
```diff
@@ -145,31 +146,31 @@ def setup_environment(self, spack_env, run_env):
         else:
             lib_dir = self.prefix.lib
 
-        run_env.set('ParaView_DIR', self.prefix)
-        run_env.set('PARAVIEW_VTK_DIR',
+        env.set('ParaView_DIR', self.prefix)
+        env.set('PARAVIEW_VTK_DIR',
                     join_path(lib_dir, 'cmake', self.paraview_subdir))
```
Comment (scheibelp) [inline var/spack/repos/builtin/packages/paraview/package.py]: @jrood-nrel Semantically the changes look good but flake is complaining about this line (need to adjust spacing). Other than that this looks good.

**Comment by scheibelp** [comment]
Thanks!

**Pull request merged**